docs/development/pr-review.asciidoc

```diff
@@ -0,0 +1,129 @@
+
+On the other hand, if a reviewer calls out a specific chunk of code as being hard to understand, then one of the first people to look at the code is having a hard time understanding it. If the reviewer feels strongly that this is a problem, they should propose alternatives for the author's consideration.
+
+When in doubt, relying on "prior art" in the codebase, especially in and around the area of the codebase that is being contributed to, is generally a good idea.
+
+There may also be times when a person is inspired by a particular contribution to introduce a new way to style code that we already have different style guidelines or "prior art" for. It's OK to bring this up in a pull request, but ultimately that discussion should branch off into a separate issue or pull request to update the appropriate guide. If this change is prompted by a reviewer, then the original PR should not be blocked on this.  If the change is prompted by the author, then they can either update the PR to be consistent with our existing guidelines (preferred) or they can choose to block the PR entirely on that separate styleguide discussion.
+
+
+[float]
+=== Nitpicking
+
+Nitpicking is when a reviewer identifies trivial and unimportant details in a pull request and asks the author to change them. This is a completely subjective category that is impossible to define universally, and it's equally impractical to define a blanket policy on nitpicking that everyone will be happy with.
+
+Reviewers should feel comfortable giving any feedback they have on a pull request regardless of how trivial it is. Authors should feel equally comfortable passing on feedback that they think is trivial and inconsequential.
+
+Often, reviewers have an opinion about whether the feedback they are about to give is a nitpick or not. While not required, it can be really helpful to identify that feedback as such, for example "nit: a newline after this would be helpful". This helps the author understand your intention.
+
+
+[float]
+=== Review feedback that is never appropriate
+
+Whether or not a bit of feedback is appropriate for a pull request is really dependent on the motivation for giving the feedback in the first place. Reviewers should not be reviewing code with the mindset of "how would I write this code?". The reviewer didn't write the code, and their goal isn't to craft the contribution into a form that is simply whatever they would have written if they had.
+
+Feedback such as "this is crap" isn't feedback at all. It's both mean and unhelpful, and it is never appropriate.
+
+
+[float]
+=== A checklist
```

Comment:
Should it mention security? Maybe some small explanation, and "When in doubt, refer the PR to someone on the #kibana-security team."?

Reply:
Let me ponder how to phrase that one. I may leave it off for now for the sake of not blocking this PR, unless I can figure out a relatively concrete way to express it besides a generic "is secure".

Reply:
I added a bullet on security concerns